modules/master-node-sizing.adoc

```diff
@@ -6,38 +6,40 @@
 [id="master-node-sizing_{context}"]
 =  Control plane node sizing
 
-The control plane node resource requirements depend on the number of nodes in the cluster. The following control plane node size recommendations are based on the results of control plane density focused testing. The control plane tests create the following objects across the cluster in each of the namespaces depending on the node counts:
+The control plane node resource requirements depend on the number and type of nodes and objects in the cluster. The following control plane node size recommendations are based on the results of a control plane density focused testing (cluster-density). This test create the following objects across a given number of namespaces.
 
-- 12 image streams
-- 3 build configurations
-- 6 builds
-- 1 deployment with 2 pod replicas mounting two secrets each
-- 2 deployments with 1 pod replica mounting two secrets
-- 3 services pointing to the previous deployments
-- 3 routes pointing to the previous deployments
-- 10 secrets, 2 of which are mounted by the previous deployments
-- 10 config maps, 2 of which are mounted by the previous deployments
+- 1 image streams
+- 1 build
+- 5 deployments with 2 pod replcias (sleep) mounting 4 secrets, 4 configmaps and 1 downwardAPI volume each
```

Comment:
replicas*
what is the context of (sleep)? Are the replica pods in a sleep state? If so, can we change this to:
- 5 deployments with 2 sleeping pod replicas mounting 4 secrets, 4 ConfigMaps, and 1 downwardAPI volume each